src/krb5_plugin/passkey/passkey_utils.c

```diff
@@ -535,7 +535,7 @@ sss_passkey_prefix_json_data(enum sss_passkey_phase phase,
     json_error_t jret;
     json_t *jroot;
     struct sss_passkey_message *message;
-    void *data;
+    void *data = NULL;
```

Comment:
Hi,

I'd like to refactor this function to avoid the covscan warning.

If I see it correctly `sss_passkey_prefix_json_data()` is only called with `phase` `sss_passkey_prefix_json_data` and `sss_passkey_reply_from_json_object()` can only return `struct  sss_passkey_reply *`. So the whole if-block at the end can be avoided.

bye,
Sumit

Reply:
Fixed. Thanks for pointing that out.